sky/tools/release_engine.py

```diff
@@ -114,8 +117,8 @@ def main():
     with open(sky_engine_revision_file, 'w') as stream:
         stream.write(commit_hash)
 
-    upload_artifacts(android_dist_root, 'android-arm', commit_hash)
```

Comment:
Can we add a step that verifies that all these paths exist before uploading any of them?  We're getting enough of them that there's a risk that the build system gets out of sync with this script and it would be nice to know that before doing a partial upload.